x-pack/plugins/cloud_security_posture/public/pages/configurations/latest_findings/latest_findings_container.tsx

```diff
@@ -107,12 +118,237 @@ const customCellRenderer = (rows: DataTableRecord[]) => ({
   ),
 });
 
+export const FINDINGS_UNIT = (totalCount: number) =>
+  i18n.translate('xpack.csp.findings.unit', {
+    values: { totalCount },
+    defaultMessage: `{totalCount, plural, =1 {finding} other {findings}}`,
+  });
+
+const defaultGroupingOptions = [
+  {
+    label: i18n.translate('xpack.csp.findings.latestFindings.groupByResource', {
+      defaultMessage: 'Resource',
+    }),
+    key: 'resource.name',
+  },
+  {
+    label: i18n.translate('xpack.csp.findings.latestFindings.groupByRuleName', {
+      defaultMessage: 'Rule name',
+    }),
+    key: 'rule.name',
+  },
+  {
+    label: i18n.translate('xpack.csp.findings.latestFindings.groupByCloudAccount', {
+      defaultMessage: 'Cloud account',
+    }),
+    key: 'cloud.account.name',
```

Comment:
Right now the grouping component aggregates all the non-matching data into this grouping with the dash "-" instead of filtering the data.

This is not technically an issue, but I'll ping @tinnytintin10 and @r4zr32d3k1l to discuss if we want our grouping component to always apply an `exists` filter to the selected group instead of showing the non-matching group.

Reply:
we decided to implement the filter in a separate PR